.changeset/eighty-roses-appear.md

```diff
@@ -0,0 +1,5 @@
+---
+"@firebase/firestore": patch
+---
+
+Add `x-goog-request-params` to REST request header.
```

Comment:
What do you think about release notes stating what is fixed in terms of what the customer understands? 

For example this could be "Fixed issue where count and lite API queries did not work with named databases."